frontend/lib/progress/progress-routes.tsx

```diff
@@ -51,6 +51,16 @@ export type ProgressRoutesProps = ProgressStepDefaults & {
    * and so on. We won't display a progress bar for these.
    */
   confirmationSteps: ProgressStepRoute[];
+
+  /**
+   * When defined, this relabels the progress bar with a new label for a certain
+   * number of steps at the start of the `stepsToFillOut` list.
+   *
+   * The two array elements are defined as follows:
+   * - first element is the label for this introductory section of the progress steps
+   * - second element is the number of steps this introductory section accounts for
+   */
+  introProgressSection?: [string, number];
```

Comment:
Unless there's a particular reason to use an array here, I'd recommend making this into an object instead:
```
{
  label: string,
  num_steps: number
}
```
That way when you access them it'll be easier to know what you're accessing.